### ISSUE TITLE
graduated prices based on product quantities

### ISSUE THREAD

**Issue by Habinho** [OFFLINE-GmbH/oc-mall-plugin#683]
Is it possible to define graduated prices based on product quantities out of the box or only by extending the plugin? If only second solution is possible, would be glad if someone could give some hints. Thx

**Comment by tobias-kuendig** [comment]
Currently, this is not possible, no. I also would have to look into the details on how to implement such a feature. Maybe you can use [October's `extend` methods](https://octobercms.com/docs/plugin/extending#extending-user-model) to override the price accessors or add a custom price method yourself, that returns the price for a given quantity.

If you find a good way of doing this using events we can always add more event calls to the core to make the plugin more extendable.

**Comment by maartenmachiels** [comment]
Hi @Habinho - I had a client that requested just this. We solved it in another way. We created a custom property named "packaging", with some options, in our case "per piece", "per five pieces", "per ten pieces"... etc. Then we checked the box "use for variants". This allowed us to use different prices for different amounts of the product. It wasn't very hard to implement this. Might this be a solution to the problem?

P.S.: We also researched the discount system to see if there might be a way to use discounts for this purpose, but this isn't available just yet.
